lib/release_notes_bot_web/controllers/slack_interaction_controller.ex

```diff
@@ -4,34 +4,32 @@ defmodule ReleaseNotesBotWeb.SlackInteractionController do
   / commands from Slack.
   """
   use ReleaseNotesBotWeb, :controller
-  alias ReleaseNotesBot.Projects
+  alias ReleaseNotesBot.{Projects, Channels}
 
   def index(conn, params) do
     %{"view" => view, "user" => user} = Projects.parse_params(params)
 
     case Projects.parse_response(view) do
       %{client: client_name, channel: channel} ->
-        Slack.Web.Chat.post_message(
```

Comment:
What happened to `Slack.Web.chat`? Is it still being used elsewhere? Can we remove it?